packaging/debs/Debian/debian/control

```diff
@@ -7,11 +7,12 @@ Uploaders: Alvaro Videla <alvaro@rabbitmq.com>,
  Jean-Sébastien Pédron <jean-sebastien@rabbitmq.com>,
  Giuseppe Privitera <giuseppe@rabbitmq.com>
 Build-Depends: debhelper (>= 9),
+ dh-systemd (>= 1.5),
```

Comment:
This breaks compatibility with older Ubuntu/Debian versions, e.g. Ubuntu 14.04. Can this be made optional?